plugins/ingest-attachment/src/test/java/org/elasticsearch/ingest/attachment/AttachmentProcessorTests.java

```diff
@@ -358,6 +358,7 @@ public void testIndexedChars() throws Exception {
 
     }
 
+    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/66086")
```

Comment:
Maybe `assumeFalse("https://github.com/elastic/elasticsearch/issues/66086", Constants.WINDOWS);` would be better, since the test only seemed to fail on windows?

Reply:
I'm going to try a fix for the test and if that doesn't work, I'll change to the `assumeFalse` above.